app/request/show/model.js

```diff
@@ -0,0 +1,13 @@
+import Model, { attr, belongsTo } from '@ember-data/model';
+
+export default class RequestModel extends Model {
+  @attr('string') id;
+  @attr('string') type;
+  @attr('string') responder;
+  @attr('string') requestor;
+  @attr('string') machine;
+  @attr('string') title;
+  @attr('string') downtime;
+  @attr('string') createdDate;
+  @belongsTo('user') requestor;
+}
```

Comment:
@kamilsobik I guess this is reason your issue - try move `model.js` to `app/request` instead `app/request/show`